rtsp.go

```diff
@@ -106,6 +106,7 @@ func init() {
 type Config struct {
 	Address        string               `json:"rtsp_address"`
 	RTPPassthrough *bool                `json:"rtp_passthrough"`
+	LazyDecode     bool                 `json:"lazy_decode"`
```

Comment:
```suggestion
	LazyDecode     bool                 `json:"lazy_decode,omitempty"`
```

Reply:
why is this needed? I think omitempty only changes the behavior if you are marshalling to json, which this struct will never do.

Reply:
Good q - eventually app/fleet want to consume these config structs to show fancy configs, so I put them in for the far flung future.